wordpress-mobile/WordPress-Android
# [Reader IA][WIP] Implements the new Reader top bar and removes tab layout and existing filter UI

Part of #19605 

This PR is one of the tasks that is part of the replacement of ViewPager by the new drop-down menu in the Reader. This PR implements the new top bar UI including the collapsing animation, search button UI and menu UI. The final menu items and icons are included.

https://github.com/wordpress-mobile/WordPress-Android/assets/14964993/8b439df5-894b-43f5-a0b7-877fa59eed33

https://github.com/wordpress-mobile/WordPress-Android/assets/14964993/799d490b-0164-4def-8ce5-5aa83c9d039d

-----

## To Test:

1 - Install Jetpack and sign-in
2 - Open the Reader: the tabs layout and filter UI should be gone, and the reader IA new top bar should be visible with the drop-down menu and the search button. You should be able to interact with the menu although the selected options will not update the reader content. This will be done in another PR. The search button click will also be handled in another PR.

-----

## Regression Notes

1. Potential unintended areas of impact
None

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual testing

3. What automated tests I added (or what prevented me from doing so)
So far only UI changes

-----

## PR Submission Checklist:


-----

## UI Changes Testing Checklist:
